examples/CMakeLists.txt

```diff
@@ -1,5 +1,41 @@
 # SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 
+cmake_minimum_required(VERSION 3.28...4.0)
+
+project(
+    beman.exemplar.examples
+    DESCRIPTION "Examaples for Beman implementation of exemplar"
+    LANGUAGES CXX
+    VERSION 2.1.1
+)
+
+if(PROJECT_IS_TOP_LEVEL)
+    if(NOT DEFINED CMAKE_CXX_STANDARD)
+        set(CMAKE_CXX_STANDARD 20)
+    endif()
+
+    find_package(beman.exemplar 2.1.2 EXACT REQUIRED)
```

Comment:
Note, the **REQUIRED** option may be removed, but to show the problem, it is set!